pysat/tests/test_instrument.py

```diff
@@ -285,6 +285,14 @@ def teardown_method(self):
         del self.testInst, self.out, self.ref_time, self.ref_doy
         return
 
+    def test_setting_data_as_tuple(self):
+        """Test setting data by name."""
```

Comment:
I'm confused by the test name and docstring, can you make them more self-consistent?